### PR TITLE
Feature/organization catalog

### LINKED ISSUE #211: Create organization catalog
Refer to organizations catalog wireframe.

### PR DESCRIPTION
**Pull request by elnzv**
Closes #211 

# Proposed changes
- organization card
- organization cards grid
- search field
- less package

### CODE REVIEW

**Comment by elnzv** [comment]
@Digipalvelutehdas/dipor-team, please review.

File: organizations/client/all/all.html [hunk 1]
```diff
@@ -1,28 +1,47 @@
...
-          <b>{{ organization.name }}</b>
-        </a>
-      </div>
-      <div class="panel-body">
-        {{ organization.description }}
+    <div class="col-md-4 col-sm-6 col-xs-12">
+      <div class="organization-card">
+        <div class="organization-logo">
+          <img class="img-circle"
+               src="http://placehold.it/70x70"
```
Comment (brylie) [inline organizations/client/all/all.html]: Lets keep a local placeholder image in our `/organizations/public/` folder, so we do not need to call a remote service.

**Comment by brylie** [comment]
I am wondering whether we should include non-functional boilerplate in this PR. E.g. we plan on adding a search feature, but the search field here does not currently work. Should the search field and organization logo placeholder be added in a different PR?

**Comment by elnzv** [comment]
@Digipalvelutehdas/dipor-team, please review.

**Comment by brylie** [comment]
@elnzv please get review from @Nazarah regarding the UX/UI.

**Comment by brylie** [comment]
@elnzv and @Nazarah, did you meet yet to discuss this pull request?

**Comment by Nazarah** [comment]
Would be better if I could see the GUI at @elnzv or @brylie 's laptop.
could not see how it is working.

**Comment by brylie** [comment]
@elnzv and @Nazarah when will you meet to discuss this UX/UI task?

**Comment by Nazarah** [comment]
Is it possible to provide option to add organization logo while adding one?
one of the client requirements were that option.

**Comment by elnzv** [comment]
@Nazarah, yeah we gonna have this feature. But since we don't have it yet, it's not included in this PR.
If everything else looks good, merge this pull request.

**Comment by brylie** [comment]
Lets take a middle ground approach here and include a static image to represent organizations within our `/organizations/public/` folder. Then, add the placeholder image to the card UI.

**Comment by brylie** [comment]
Make sure the placeholder image is:
- relevant
- relatively nice
- freely licensed (e.g. public domain)
  - important: verify the license and include attribution if required

**Pull request merged**